--- src/bin/yasweb.rs
+++ src/bin/yasweb.rs
@@ -957,11 +957,11 @@
         },
         AppArg {
             name: "--type".to_string(),
-            arg_type: ArgType::String,
+            arg_type: ArgType::List,
             required: false,
             default_value: None,
-            options: None,
+            options: Some(vec!["".to_string(), "standard report".to_string(), "report manager".to_string()]),
             depends_on: None,
             autofill: if type_autofills.is_empty() {
                 None
             } else {
