--- src/bin/yasweb.rs
+++ src/bin/yasweb.rs
@@ -876,14 +876,26 @@
                 })
                 .collect();
         }
-        args.push(AppArg {
-            name: "--type".to_string(),
-            arg_type: ArgType::String,
-            required: false,
-            default_value: None,
-            options: None,
-            depends_on: None,
-            autofill: None,
-        });
+        args.push(AppArg {
+            name: "--type".to_string(),
+            arg_type: ArgType::List,
+            required: false,
+            default_value: None,
+            options: Some(vec![
+                "".to_string(),
+                "standard".to_string(),
+                "report_manager".to_string(),
+            ]),
+            depends_on: None,
+            autofill: None,
+        });

         args.push(AppArg {
